### PR TITLE
Queue API Implementation (AWS SQS)

### PR DESCRIPTION
**Pull request by ggallotti**
Java Implementation for the Queue API. 

Issue: 96582

### CODE REVIEW

File: common/src/main/java/com/genexus/ModelContext.java [hunk 1]
```diff
@@ -43,7 +43,9 @@ public static ModelContext getModelContext()
         ModelContext context = (ModelContext)threadModelContext.get();
 		if(context == null)
 		{
-			logger.error(new Date() + " - Cannot find ModelContext for thread " + Thread.currentThread() );
+			if (logger.isDebugEnabled()) {
```
Comment (iroqueta) [inline common/src/main/java/com/genexus/ModelContext.java:46]: It's not necessary to ask for isDebugEnabled.

Comment (iroqueta) [reply common/src/main/java/com/genexus/ModelContext.java:46]: The debug method already have the if statement

	public void debug(String msg) {
		if (log.isDebugEnabled()) {
			log.debug(msg);
		}
	}

I understand the performance issue, but it seems that is a redundant code

Comment (ggallotti) [reply common/src/main/java/com/genexus/ModelContext.java:46]: Ready, removed "if (log.isDebugEnabled()) {" from "public void debug(String msg)" method.

File: gxqueueazure/pom.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    <parent>
+        <artifactId>parent</artifactId>
+        <groupId>com.genexus</groupId>
+        <version>2.7-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>gxqueueazure</artifactId>
+    <dependencies>
+        <dependency>
```
Comment (iroqueta) [inline gxqueueazure/pom.xml:14]: This dependency seems to not be necessary

Comment (ggallotti) [reply gxqueueazure/pom.xml:14]: Fixed!

**Review by iroqueta — APPROVED** [review]

**Review by iroqueta — APPROVED** [review]

**Pull request merged**